README.md

```diff
@@ -1,9 +1,3 @@
-# News API
+# Northcoders News API
 
-If you would like to run this project you will need to clone the repository and add two .env files with the daatabase file names
-
-.env.development
-    PGDATABASE=nc_news
-
-.env.test
-    PGDATABASE=nc_news_test
\ No newline at end of file
+For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
```

Comment:
I should be able to see the stuff you have deleted - ideally you should be explaining what a dev has to do in order to use this repo